auth-tokens-filter/src/main/java/com/palantir/tokens/auth/http/BearerTokenLoggingContextFilter.java

```diff
@@ -65,6 +65,14 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
                     // slf4j
                     MDC.put("userId", jwt.getUnverifiedUserId());
 
+                    if (jwt.getUnverifiedSessionId().isPresent()) {
```

Comment:
nit: let's keep these three together without line breaks (so that they all belong to the `//slf4j` comment)